src/components/function-summary/value.tsx

```diff
@@ -77,24 +77,33 @@ const TupleResult = ({ tuple, isPoxAddr, btc }: any) => {
   );
 };
 
-const getValue = (arg: { name: string; type: any; repr: any; value: any }, btc: null | string) => {
+const isJSONString = (str: string) => {
+  try {
+    JSON.parse(str);
+    return true;
+  } catch (error) {
+    return false;
+  }
+};
+
+const getValue = (arg: any, btc: null | string) => {
   if (arg.type === 'uint') {
-    const value = arg.repr.replace('u', '');
-    if (arg.name.includes('ustx')) {
+    const value = arg?.repr?.replace('u', '');
+    if (arg?.name?.includes('ustx')) {
```

Comment:
I suggest we remove the optional operator `?` whenever possible. It looks like it used to expect arg to always exist.